scripts/exglobal_atmos_analysis.sh

```diff
@@ -925,8 +925,6 @@ OBS_INPUT::
    sstviirs       viirs-m     npp         viirs-m_npp         0.0     4     0
    sstviirs       viirs-m     j1          viirs-m_j1          0.0     4     0
    ahibufr        ahi         himawari9   ahi_himawari9       0.0     1     0
-   atmsbufr       atms        n21         atms_n21            0.0     1     1
```

Comment:
The atms_n21 and cris-fsr_n21 lines were removed from `exglobal_atmos_analysis.sh` via PR #611.   Why are we seeing them removed again?

A check of [ADCollard:release/gfsda.v16](https://github.com/ADCollard/GSI/tree/release/gfsda.v16) shows 
 the branch is 1 commit behind [NOAA-EMC:release/gfsda.v16](https://github.com/NOAA-EMC/GSI/tree/release/gfsda.v16).  

We should update ADCollard:release/gfsda.v16 with the current head of NOAA-EMC/release/gfsda.v16.  If we do so the change to `exglobal_atmos_analysis.sh` in this PR should disappear.